src/extra/AutoUI/Filters/PersistentFilters.tsx

```diff
@@ -148,11 +144,8 @@ export const PersistentFilters = ({
 	// and we were able to load some from the local storage, then communicate them
```

Comment:
```suggestion
	// and we were able to load some from the page url, then communicate them
```
but the whole comment might need changes.